doc/source/getting_started/index.rst

```diff
@@ -3,65 +3,65 @@
 Getting Started
 ===============
 
-Please, carefully read all the instructions in this page in order to achieve a
-successfully install of ``ansys-templates``.
+To successfully install ``pyansys-templates``, carefully read all instructions on this page.
```

Comment:
Although the repository is named `pyansys-templates`, the tool provided is named `ansys-templates`.

```suggestion
To successfully install ``ansys-templates``, carefully read all instructions on this page.
```